src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurRequestExtractRequestDetailFileLineFixture.java

```diff
@@ -11,23 +11,26 @@
 import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
 
 public enum ConcurRequestExtractRequestDetailFileLineFixture {
+
+    REQUEST_DETAIL_GOOD_LINE_1((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YA"), new KualiDecimal(10.00), new String("CORNELL"), 1),
+    REQUEST_DETAIL_GOOD_LINE_2((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YE"), new KualiDecimal(40.00), new String("CORNELL"), 2),
+    REQUEST_DETAIL_GOOD_LINE_3((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YH"), new KualiDecimal(200.00), new String("CORNELL"), 1),
+    REQUEST_DETAIL_GOOD_LINE_4((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YK"), new KualiDecimal(3.33), new String("CORNELL"), 3),
+    REQUEST_DETAIL_GOOD_LINE_5((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YN"), new KualiDecimal(5000.00), new String("CORNELL"), 1),
     
-    REQUEST_DETAIL_LINE_1((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YA"), new KualiDecimal(10.00), 1),
-    REQUEST_DETAIL_LINE_2((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YE"), new KualiDecimal(40.00), 2),
-    REQUEST_DETAIL_LINE_3((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YH"), new KualiDecimal(200.00), 1),
-    REQUEST_DETAIL_LINE_4((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YK"), new KualiDecimal(3.33), 3),
-    REQUEST_DETAIL_LINE_5((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YN"), new KualiDecimal(5000.00), 1);
-    
-    
+    REQUEST_DETAIL_BAD_EMPLOYEE_GROUP_ID_LINE_2((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YE"), new KualiDecimal(40.00), new String("HARVARD"), 2);
+
     public final Date batchDate;
```

Comment:
Java's SQL Date objects are mutable. If you want your fixture objects to be fully immutable, I would suggest storing either a Joda DateTime or a date String on the fixture, and then converting to a SQL Date when creating the actual detail line.